### DISCUSSION TITLE
Add clean cli command

### DISCUSSION THREAD

**Discussion by roberthgnz** [vercel/next.js#57193] [Ideas]
### Goals

1. Remove generated files and caches.

### Background

When you are in development, you install multiple packages, make version changes etc. Many of the above remain in cache and produce errors.

### Proposal

Adding it to the next cli as a new option.

**Comment by gavenkoa** [comment]
see https://github.com/vercel/next.js/discussions/6009 & https://nextjs.org/docs/pages/building-your-application/upgrading/version-11
